src/MainWindow.xaml.cs

```diff
@@ -304,6 +317,14 @@ void ProcessCommandLineArguments()
                 return;
             }
 
+            if (args[1].ToLower() == "-mcpserver")
```

Comment:
Rather than assume that there will only be one arg on the command line, at arg[1], loop through the args to allow for either -lazy or -mcpserver, in either order.